bin/tag-release.js

```diff
@@ -1,72 +1,47 @@
 #!/usr/bin/env node
 /* eslint no-console: 0 */
 const commander = require("commander");
-const _ = require("lodash");
-const utils = require("../src/utils.js");
-const help = require("../src/help");
 const pkg = require("../package.json");
 const path = require("path");
-const logger = require("better-console");
-const chalk = require("chalk");
-
-const api = require("../src/index.js");
-
-const questions = {
-	github: [
-		{
-			type: "input",
-			name: "username",
-			message: "What is your GitHub username"
-		},
-		{
-			type: "password",
-			name: "password",
-			message: "What is your GitHub password"
-		}
-	]
-};
+const help = require("../src/help");
+const utils = require("../src/utils.js");
 
 commander
 	.version(pkg.version)
-	.option(
-		"-r, --release [type]",
-		"Release type (major, minor, patch, premajor, preminor, prepatch, prerelease)",
-		/^(major|minor|patch|premajor|preminor|prepatch|prerelease)/i
-	)
-	.option(
-		"-c, --config <filePath>",
-		"Path to JSON Configuration file (defaults to './package.json')",
+	.option("--verbose", "Console additional information", { noHelp: true })
```

Comment:
I wonder if we should list `--maxbuffer` here although I know it doesn't show itself on sub-commands which is why you provide it in the `bin` folder, but by putting it here I think it'll show up when you do `tag-release --help`

Reply:
It will show up if we do `tag-release --help`, however it will break our running since `tag-release --verbose` would be valid and wouldn't launch any sub-commands so I would have to handle that within the `tag-release.js` file, also it wouldn't run `start` since an option is provided it isn't going to use the default.